Makefile

```diff
@@ -7,11 +7,16 @@ SPHINXOPTS    ?=
 SPHINXBUILD   ?= sphinx-build
 SOURCEDIR     = docs
 BUILDDIR      = docs/_build
+INITSCRIPT	  = initial_setup.sh
 
 # Put it first so that "make" without argument is like "make help".
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
+# Initial setup
+init:
+	@sudo bash "$(INITSCRIPT)"
```

Comment:
I think `sudo` is not a good idea here. Docker can be setup to not need superuser rights (that's how I set it up, for example), and even if it's not, it's better if the user prepends his script/make call with a `sudo` himself. `docker-compose` will tell the user if he doesn't have sufficient permissions.

Reply:
Yeah, I was confused on this one. Thanks for clearing it out for me. Will do that.